docs/UsersGuide/source/Quickstart.rst

```diff
@@ -212,6 +228,8 @@ For automatic resubmission of the workflow (every 3 minutes), one of the followi
 to the user's crontab (use ``crontab -e`` to edit the cron table) depending on your preference of how 
 you call Rocoto:
 
+NOTE DOM 20220227 rocotorun will not be in the path unless the user sets PATH=... in the crontab
+
 .. code-block:: console
```

Comment:
The goal of the launch script is so the user doesn't need to worry about whether rocotorun is in their path when running from the command line.  In addition, if the user sets USE_CRON_TO_RELAUNCH in the config.sh, they don't need to worry about setting anything in the cron, since the cron points to the launch script (which loads the Rocoto module).

Reply:
To avoid confusion here, I will just list */3 * * * * cd /glade/p/ral/jntp/$USER/expt_dirs/test_CONUS_25km_GFSv15p2 && ./launch_FV3LAM_wflow.sh as the only option for cron here. That should avoid this problem.